packages/eas-cli/src/credentials/ios/appstore/experimental.ts

```diff
@@ -1,7 +1,4 @@
 import { BundleId, Profile, RequestContext } from '@expo/apple-utils';
-import { boolish } from 'getenv';
```

Comment:
maybe remove dep?

Reply:
I already checked, but it's used elsewhere now as well.